### DISCUSSION TITLE
Request to revert to the specification allowing the selection of Amazon S3 connection ports

### DISCUSSION THREAD

**Discussion by ky-iwanami** [iterate-ch/cyberduck#15579] [Q&A] [answered]
Our development team is currently conducting local testing of Amazon S3 using LocalStack.
We have been using port 4566 for this purpose. 
However, with the recent update, the specification allowing the flexible selection of ports for Amazon S3 has been changed to a fixed port 443, causing complications in our environment setup. 
Specifically, depending on the version of CyberDuck, we are now required to adjust the LocalStack port accordingly.
Therefore, we kindly request that the specification allowing the selection of ports be reinstated.

**Comment by dkocher** [comment, marked as answer]
Changed with #15363. Please use the generic S3 connection profile _S3 (HTTPS)_ available in _Preferences → Profiles_.

**Comment by ky-iwanami** [comment]
Thank you, the issue has been resolved.
